scripts/globals/weaponskills/metatron_torment.lua

```diff
@@ -39,7 +39,7 @@ function onUseWeaponSkill(player, target, wsID, tp, primary, action, taChar)
 
     local damage, criticalHit, tpHits, extraHits = doPhysicalWeaponskill(player, target, wsID, params, tp, action, primary, taChar)
     if damage > 0 then
-        if not target:hasStatusEffect(tpz.effect.ATTACK_DOWN) then
+        if not target:hasStatusEffect(tpz.effect.DEFENSE_DOWN) then
```

Comment:
it doesn't even need to check - defense down naturally doesn't reapply unless the new effects power is greater than the existing effect.

Reply:
Thank you - I'm ridiculously new at this so forgive the question - does that mean i should just delete the if not statement entirely as it is checking that elsewhere? REmoving it is of some benefit since it was checking attack_down, right?

Reply:
yes you want that block to become:
```lua
    if damage > 0 then
        local duration = tp / 1000 * 20 * applyResistanceAddEffect(player, target, tpz.magic.ele.WIND, 0)
        target:addStatusEffect(tpz.effect.DEFENSE_DOWN, 19, 0, duration)

        -- Apply aftermath
        tpz.aftermath.addStatusEffect(player, tp, tpz.slot.MAIN, tpz.aftermath.type.RELIC)
    end
```
note the adjusted indent on the part that was inside the if-then-end pairing

Reply:
cool, thanks i changed and committed again so hopefully that is right.

Reply:
looks good :+1: